packages/access-client/src/utils/did-mailto.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {string} email
+ * @returns {`did:mailto:${string}:${string}`}
+ */
+export function createDidMailtoFromEmail(email) {
```

Comment:
could you please export this from the package so I can use it in w3ui?

Reply:
pushed to our feature branch as https://github.com/web3-storage/w3protocol/pull/433/commits/0cc4e47fd8383927393c32e7d06fc5e56f0fe10a